### PR TITLE
Work and about self feature

### PR DESCRIPTION
**Pull request by George7h**
- Add .html file
- Add .css file
- upload images
- added work section
- added about me section

### CODE REVIEW

File: HTML&CSS/Website.html [hunk 1]
```diff
@@ -38,5 +38,128 @@ <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
...
+    <ul class="languages">
+        <h3 class="h3">Languages</h3><a href="#"><img src="../HTML&CSS/icons/Disabled.png"></a>
+        <li class="langlist langlist1"><img src="../HTML&CSS/Images/skill list/js.svg"><a href="#">JavaScript</a></li>
+        <li class="langlist langlist2"><img src="../HTML&CSS/Images/skill list/html.svg"><a href="#">HTML</a></li> 
+        <li class="langlist langlist3"><img src="../HTML&CSS/Images/skill list/css.svg"><a href="#">CSS</a></li>  
+    </ul>
+    <ul class="Frameworks"><h3 class="h3">Frameworks</h3></ul>
+    <ul class="Skills"><h3 class="h3">Skills</h3></ul>
+    </ul>
+</div>
```
Comment (Alfredbis29) [inline HTML&CSS/Website.html]: - [ ]I can see that you have developed a nice design thus far, but there is still the potential for improvement. I noticed that your design does not match the Figma design; I recommend that you use the Figma design for your project to make it look more professional. :smile:  Put some space between your `headers`
Your Design :wink: 
Screenshot from 2023-06-03 20-23-41

Figma Design 
Screenshot from 2023-06-03 20-19-00

File: HTML&CSS/Website.html [hunk 2]
```diff
@@ -38,5 +38,128 @@ <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
...
+    <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
+    <ul class="social-media-icon">
+        <li><a class="icon" href="https://twitter.com/GJ2H"><img src="../HTML&CSS/icons/Twitter.svg"></a></li>
+        <li><a class="icon" href="https://linkedin.com/in/george-hamman-95b98224b"><img src="../HTML&CSS/icons/Linkedin icon.svg"></a></li> 
+        <li><a class="icon" href="https://github.com/George7h"><img src="../HTML&CSS/icons/GIT.svg"></a></li> 
+        <li><a class="icon" href="#"><img src="../HTML&CSS/icons/M.svg"></a></li> 
+        <li><a class="icon" href="#"><img src="../HTML&CSS/icons/Piece.svg"></a></li> 
+    </ul>
+    <button class="resume" href="#">Get my resume</button>
+</div>
```
Comment (Alfredbis29) [inline HTML&CSS/Website.html]: - [x] Please follow the Figma design. As you can see, the about session does not fit the Figma design. If we compare your design to the Figma design, it is somewhat different. Please change this. :point_down: 

Your Design 
Screenshot from 2023-06-03 20-31-39

Figma Design 
Screenshot from 2023-06-03 20-31-13

File: HTML&CSS/Website.html [hunk 3]
```diff
@@ -38,5 +38,128 @@ <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
...
+
+<div class="containing">
+    <ul class="skill-list">
+    <ul class="languages">
+        <h3 class="h3">Languages</h3><a href="#"><img src="../HTML&CSS/icons/Disabled.png"></a>
+        <li class="langlist langlist1"><img src="../HTML&CSS/Images/skill list/js.svg"><a href="#">JavaScript</a></li>
+        <li class="langlist langlist2"><img src="../HTML&CSS/Images/skill list/html.svg"><a href="#">HTML</a></li> 
+        <li class="langlist langlist3"><img src="../HTML&CSS/Images/skill list/css.svg"><a href="#">CSS</a></li>
```
Comment (Alfredbis29) [inline HTML&CSS/Website.html]: - [x]  The arrow down the sign on Your Languages session is not properly fixed, making the design show up unprofessional; even the colors and fonts are not exactly the same as the Figma design; please match your design to figma one as necessary for this milestone of your project.
Your design 
Screenshot from 2023-06-03 20-36-23

Figma Design 
Screenshot from 2023-06-03 20-36-00

File: HTML&CSS/Website.html [hunk 4]
```diff
@@ -38,5 +38,128 @@ <h2 class="connect"><a href="#">LET'S CONNECT</h2></a>
...
+            <div class="post4">
+                <img class="multi-post-image" src="../HTML&CSS/Images/arti.svg">
+            <h2 class="post-title">Multi-Post Stories</h2>
+                <div>
+                    <ul class="post-subtitle">
+                        <li class="client">Canope</li>
+                        <li class="role">Back End Dev</li>
+                        <li class="year">2015</li>
+                    </ul>
+                </div>
```
Comment (Alfredbis29) [inline HTML&CSS/Website.html:112]: - [x]  Your design is wonderful, but let's make it more professional by following the figma design guidelines; I can see where you need to improve:point_down:  
You should begin with 'HTML','CSS','JAVASCRIPT', and so forth. Please arrange them as they are on the figma design, and fix the padding on the show project button, which is not the same as the original design. :wink: 

Figma Design 
Screenshot from 2023-06-03 20-43-08

Your Design 
Screenshot from 2023-06-03 20-42-38

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @George7h 
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
:heavy_check_mark:  Your Work looks Professional 
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Descriptive PR
 
 ## Required Changes 
- [x] Check under the review

 ## Optional suggestions 
N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @George7h,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**